### PR TITLE
Add Azerbaijani language support

### PR DESCRIPTION
**Pull request by vusallyv**
### All Submissions:

- [x] Are tests passing? (From the root-level of the repository please run `pip install tox && tox`)
- [x] I have added or updated a test to cover the changes proposed in this Pull Request
- [x] I have updated the documentation to cover the changes proposed in this Pull Request

### CODE REVIEW

File: rosetta/locale/az/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+"your settings file should fix this."
+msgstr ""
+
+#: rosetta/templates/rosetta/file-list.html:14
+msgid "Filter"
+msgstr "Filtr"
+
+#: rosetta/templates/rosetta/file-list.html:15
+msgid "Project"
+msgstr "Proyekt"
```
Comment (nemoralis) [inline rosetta/locale/az/LC_MESSAGES/django.po]: We have "layihə" in Azerbaijani for this word. "Proyekt" is not a common (ümumişlək) word

File: rosetta/locale/az/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+msgid "All"
+msgstr "Hamısı"
+
+#: rosetta/templates/rosetta/file-list.html:31
+msgid "Application"
+msgstr "Tətbiq"
+
+#: rosetta/templates/rosetta/file-list.html:32
+msgid "Progress"
+msgstr "Proqress"
```
Comment (nemoralis) [inline rosetta/locale/az/LC_MESSAGES/django.po]: This should be "Proqres"

File: rosetta/locale/az/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+msgstr "Başqa bir fayl seçin"
+
+#: rosetta/templates/rosetta/form.html:9
+msgid "Download this catalog"
+msgstr "Bu kataloqu yükləyin"
+
+#: rosetta/templates/rosetta/form.html:24
+#, python-format
+msgid "Progress: %(percent_translated)s%%"
+msgstr "ProqressI %(percent_translated)s%%"
```
Comment (nemoralis) [inline rosetta/locale/az/LC_MESSAGES/django.po]: This should be "Proqres"

**Review by nemoralis — CHANGES_REQUESTED** [review]

**Comment by vusallyv** [comment]
@mbi could you please review this PR too?

**Comment by mbi** [comment]
I was kinda hoping you would address @nemoralis' comments above. I can't judge how pertinent they are and was hoping you guys come to an agreement on what the better translations would be? :pray:

**Comment by vusallyv** [comment]
I have corrected the translations based on the change requests of @nemoralis.

**Comment by mbi** [comment]
Ah yes, so you did, thank you!

**Comment by vusallyv** [comment]
Welcome!

**Pull request merged**